### PR TITLE
feat: deprecate ropsten and rinkeby

### PR DESCRIPTION
**Pull request by isaackps**
## Summary

Deprecating Ropsten and Rinkeby Testnets.

_On Standby for 27 Oct_

## Changes

- Deprecate Ropsten and Rinkeby Testnets
- Make Goerli testnet default

## Issues

https://www.pivotaltracker.com/n/projects/2424361/stories/183529907

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,7 +30,7 @@
     "integration:headful": "concurrently -k -s first \"npm:blockchain\" \"npm:integration:local\" \"npm:integration:metamask\"",
     "lint": "eslint ./src --ext .js,.ts,.tsx --max-warnings 0",
     "lint:fix": "npm run lint -- --fix",
-    "prep": "run-s lint:fix test integration:headless integration:headful",
+    "prep": "run-s lint:fix test integration:headless:dev integration:headful",
```
Comment (superical) [inline package.json:33]: Just curious, what's the difference in the test when running `integration:headless` and `integration:headless:dev` for the `prep` script?

**Review by superical — APPROVED** [review]
Looks good! I think you have covered all the places that still use ropsten and rinkeby in frontend. I wonder if there may be any react components in that UI component repo that are defaulted to ropsten or rinkeby and are used here? 

Thanks for preparing this PR for the deprecation of those networks!

File: package.json (hunk 1 above)
Comment (isaackps) [reply package.json:33]: not too sure but the config different for `:dev` and `:prod`. u think should change to prod instead?

Comment (simboonlong) [reply package.json:33]: https://github.com/TradeTrust/tradetrust-website/pull/628

Comment (superical) [reply package.json:33]: Ahh... Okay, then having `dev` here as what @isaackps has it seems to make sense to me.👍

**Comment by superical** [comment]
This PR will need to update the new version of that UI components report after https://github.com/TradeTrust/tradetrust-ui-components/pull/141 has been merged.

File: package.json (hunk 1 above)
Comment (simboonlong) [reply package.json:33]: `prep` was a naive task i added in long time back to run through all “ci” tasks before committing.

never really used it anymore. is can remove one.

Comment (isaackps) [reply package.json:33]: actually i still use it leh, cos i feel that it is very useful. I usually run prep 1st before pushing it up.

**Comment by isaackps** [comment]
> This PR will need to update the new version of that UI components report after [TradeTrust/tradetrust-ui-components#141](https://github.com/TradeTrust/tradetrust-ui-components/pull/141) has been merged.

Yup, so i was thinking actually we can merge UI component now, since most of it is just for the UI storybook cases.

@superical What do u think? ok to merge UI component now?

**Comment by superical** [comment]
> > This PR will need to update the new version of that UI components report after [TradeTrust/tradetrust-ui-components#141](https://github.com/TradeTrust/tradetrust-ui-components/pull/141) has been merged.
> 
> Yup, so i was thinking actually we can merge UI component now, since most of it is just for the UI storybook cases.
> 
> @superical What do u think? ok to merge UI component now?

Do we expect any new PR merges in the time being until our target date? The UI components PR looks to me will be merged as a minor revision which a new deployment by the frontend will pull into it.

**Comment by isaackps** [comment]
> > > This PR will need to update the new version of that UI components report after [TradeTrust/tradetrust-ui-components#141](https://github.com/TradeTrust/tradetrust-ui-components/pull/141) has been merged.
> > 
> > 
> > Yup, so i was thinking actually we can merge UI component now, since most of it is just for the UI storybook cases.
> > @superical What do u think? ok to merge UI component now?
> 
> Do we expect any new PR merges in the time being until our target date? The UI components PR looks to me will be merged as a minor revision which a new deployment by the frontend will pull into it.

Yup, thats y i created it as a `fix`.
i not to sure about this repo, but from the looks of it no urgent pr ba.

**Pull request merged**